junit-platform-launcher/src/main/java/org/junit/platform/launcher/LauncherConstants.java

```diff
@@ -164,6 +164,59 @@ public class LauncherConstants {
 	@API(status = EXPERIMENTAL, since = "1.10")
 	public static final String DRY_RUN_PROPERTY_NAME = "junit.platform.execution.dryRun.enabled";
 
+	/**
+	 * Property name used to enable stack trace pruning.
```

Comment:
Reads a bit odd with the sentence below.

```suggestion
	 * Property name used to enable or disable stack trace pruning.
```